devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -149,9 +144,9 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 	def results = []
 	if (cmd.notificationType == 0x07) {          // Burglar
 		if (cmd.event == 0x08) {                 // detected
-			results += sensorMotionEvent(1)
+			results << sensorMotionEvent(1)
```

Comment:
Since `sensorMotionEvent` returns a list, it would be correct to leave this as `+=`, otherwise you'd need to flatten the resulting list.

Same with Line 149.